tests/Unit/Utilities/Test_StdArrayHelpers.cpp

```diff
@@ -151,3 +152,30 @@ SPECTRE_TEST_CASE("Unit.Utilities.StdArrayHelpers.map_array",
         return x;
       }) == std::array<int, 1>{{4}});
 }
+
+namespace {
```

Comment:
Can you run this through clang-format? We use Google style not GNU :)

Reply:
Understood. Is it recommended to pass all new code through clang-format, or is that potentially hazardous?

Reply:
Ideally with `git clang-format -f` so that you only format the lines of code you edited rather than the whole file. This reduces noise but also means the code conforms to our style. You should be able to also set up your editor to use the google style by default (I know Emacs has a package and I expect vim to have one as well)